Components/Image/image.html

```diff
@@ -0,0 +1,112 @@
+        </div>
+    </header>
+    <main>
+        <div class="side-bar">
+            <ul>
+                <li>
+                    <h3>Get-started</h3>
+                </li>
+                <hr>
+                <li><a class="sidebar-item" href="/Get-Started/introduction.html">Introduction</a></li>
+                <li><a class="sidebar-item " href="/Get-Started/typography.html">Typography</a></li>
+                <li><a class="sidebar-item" href="">Colours</a></li>
+                <li>
+                    <h3>Components</h3>
+                </li>
+                <hr>
+                <li><a class="sidebar-item" href="/Components/Avatar/avatar.html">Avatar</a></li>
+                <li><a class="sidebar-item " href="/Components/Alert/alert.html">Alert</a></li>
+                <li><a class="sidebar-item " href="/Components/Badge/badge.html">Badge</a></li>
+                <li><a class="sidebar-item " href="/Components/Button/button.html">Button</a></li>
+                <li><a class="sidebar-item " href="/Components/Card/card.html">Card</a></li>
+                <li><a class="sidebar-item link-active" href="#">Image</a></li>
+                <li><a class="sidebar-item" href="/Components/Input/input.html">Input</a></li>
+                <li><a class="sidebar-item" href="/Components/Slider/slider.html">Slider</a></li>
+                <li><a class="sidebar-item" href="/Components/Grid/grid.html">Grid</a></li>
+                <li><a class="sidebar-item" href="/Components/Navigation/navigation.html">Navigation</a></li>
+            </ul>
+        </div>
+        <section class="image-section">
+            <div class="section-title">
+                Image
+            </div>
+            <hr>
+            <article>
+                <div class="article-title">
+                    Responsive Image
+                </div>
+                <div class="article-subtitle">
+                    Responsive Image Components are used to display Images on websites, which are screen responsive.
+                    They expand upto the width of the container they are in. To use this image component, name the class
+                    as <span>responsive-image</span> for your image element.
+                </div>
+                <div class="responsive-image-container">
+                    <img class="responsive-image" src="/Assets/Images/Forest-landscape.jpg" />
+                </div>
+                <div class="code-frame">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28184%2C176%2C141%2C0%29&t=3024-night&wt=none&l=auto&width=680&ds=true&dsyoff=0px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=4x&wm=false&code=%2520%253Cimg%2520class%253D%2522responsive-image%2522%2520src%253D%2522image-path-link%2522%252F%253E"
+                        style="width: 100%; height: 190px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
you should avoid inline styles

Reply:
Yeah, I've avoided them. Thanks for the suggestion 👍🏻